.github/workflows/build.yml

```
@@ -45,3 +45,42 @@ jobs:
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: 'npm'
+      - name: Get baseline branch
+        run: |
+          # Get the base branch (main) to compare against
+          echo "BASE_BRANCH=${{ github.base_ref }}" >> $GITHUB_ENV
+      - run: npm ci
+      - run: npm run build
+        # Checkout base branch for baseline
```

YAMLlint (1.38.0)

[warning] comment not indented like content

(comments-indentation)